_plugins/docs_generator.rb

```diff
@@ -64,6 +64,9 @@ def generate(site)
       documents = {}
       repo_build['documents'].each do |permalink, local_content|
         parent_path, * = permalink.rpartition('/')
+        while not parent_path.empty? and not documents.key? parent_path
+          parent_path, * = parent_path.rpartition('/')
```

Comment:
Minor nitpick but I think this is clearer when the tail isn't needed.
```suggestion
          parent_path = parent_path.rpartition('/').first
```

Reply:
Neat! Suggestion accepted in 386f8b65f6a1fb6fd758cf0c5309cbf5b7a06c77 (to not mess with the DCO).